SConstruct

```diff
@@ -351,7 +353,10 @@ def scons(): # pylint: disable=too-many-locals
 
     prereqs = PreReqComponent(env, opts, commits_file)
     if not GetOption('help') and not GetOption('clean'):
-        daos_build.load_mpi_path(env)
+        mpicc = WhereIs('mpicc')
+        if mpicc:
+            env.PrependENVPath('PATH', os.path.dirname(mpicc))
```

Comment:
(pylint-no-member) Instance of 'DefaultEnvironment' has no 'PrependENVPath' member